docs/7.0. Measurement_units_guidelines.md

```diff
@@ -0,0 +1,32 @@
+# Measurement units guidelines
+
+_(c) AMWA 2018, CC Attribution-ShareAlike 4.0 International (CC BY-SA 4.0)_
+
+This document describes the guidelines on how to use measurement units when defining a `number/measurement` event_type.
+
+Other sections can be accessed from the [Overview](1.0.%20Overview.md).
+
+## 1. Introduction
+
+The aim of this document is to create a set of guidelines on how to use measurement units in the Event & Tally specification with the aim of increasing compatibility between implementations.
+
+## 2. Guidelines
+
+### 2.1. Case sensitive behaviour
+
+Units of measurement can be case sensitive and this behaviour is allowed.
+
+### 2.2 Recommended symbols
+
+It is recommended to use the SI symbol associated with the unit of measurement whenever possible, with the following exceptions:
+
+* if the symbol is outside of the English alphabet use the English literal name (e.g. for resistance use `ohm`)
+* for degrees Celsius use `C` without the degrees symbol
+
+### 2.3 Prefixes
+
+The use of prefixes is allowed but discouraged apart from SI base units (e.g. `kg`).
+The following further remarks apply to prefixes:
+
+* micro (10^6) will be prefixed with `u`
+* `k` prefix will always stand for 1000 units and not 2^10 (1024); for the latter use `kB`
```

Comment:
I would mandate that k, M, G etc. are always power of ten, even for bytes and bits as using one notation is the only way to achieve consistency.
Also, we should add M and G and they maybe just say etc.